README.md

```diff
@@ -159,140 +244,140 @@ moviedb.searchMovie(parameters).then(...)
+| tvAccountStates         |
+| tvAlternativeTitles     |
+| tvChanges               |
+| tvContentRatings        |
+| tvCredits               |
+| episodeGroups           |
+| tvExternalIds           |
+| tvImages                |
+| tvKeywords              |
+| tvRecommendations       |
+| tvReviews               |
+| tvScreenedTheatrically  |
+| tvSimilar               |
+| tvTranslations          |
+| tvVideos                |
+| tvWatchProviders        |
+| tvRatingUpdate          |
+| tvRatingDelete          |
+| tvLatest                |
+| tvAiringToday           |
+| tvOnTheAir              |
+| tvPopular               |
+| tvTopRated              |
+| seasonInfo              |
+| seasonChanges           |
+| seasonAccountStates     |
+| seasonCredits           |
+| seasonExternalIds       |
+| seasonImages            |
+| seasonVideos            |
+| episodeInfo             |
+| episodeChanges          |
+| episodeAccountStates    |
+| episodeCredits          |
+| episodeExternalIds      |
+| episodeImages           |
+| episodeTranslations     |
+| episodeRatingUpdate     |
+| episodeRatingDelete     |
+| episodeVideos           |
+| personInfo              |
+| personChanges           |
+| personMovieCredits      |
+| personTvCredits         |
+| personCombinedCredits   |
+| personExternalIds       |
+| personImages            |
+| personTaggedImages      |
+| personTranslations      |
+| personLatest            |
+| personPopular           |
+| creditInfo              |
+| listInfo                |
+| listStatus              |
+| createList              |
+| createListItem          |
+| removeListItem          |
+| clearList               |
+| deleteList              |
+| genreMovieList          |
+| genreTvList             |
+| keywordInfo             |
+| keywordMovies           |
+| companyInfo             |
 | companyAlternativeNames |
-| companyImages |
-| accountInfo |
-| accountLists |
-| accountFavoriteMovies |
-| accountFavoriteTv |
-| accountFavoriteUpdate |
-| accountRatedMovies |
-| accountRatedTv |
-| accountRatedTvEpisodes |
-| accountMovieWatchlist |
-| accountTvWatchlist |
-| accountWatchlistUpdate |
-| changedMovies |
-| changedTvs |
-| changedPeople |
-| movieCertifications |
-| tvCertifications |
-| networkInfo |
+| companyImages           |
+| accountInfo             |
+| accountLists            |
+| accountFavoriteMovies   |
+| accountFavoriteTv       |
+| accountFavoriteUpdate   |
+| accountRatedMovies      |
+| accountRatedTv          |
+| accountRatedTvEpisodes  |
+| accountMovieWatchlist   |
+| accountTvWatchlist      |
+| accountWatchlistUpdate  |
+| changedMovies           |
+| changedTvs              |
+| changedPeople           |
+| movieCertifications     |
+| tvCertifications        |
+| networkInfo             |
 | networkAlternativeNames |
-| networkImages |
-| review |
-| episodeGroup |
+| networkImages           |
+| review                  |
+| episodeGroup            |
 
```

Comment:
Originally a script generated the table for me, but after migrating to TS I don't have it anymore. While it looks better when viewing the raw readme, the markdown parser formats the table correctly regardless of the pipes lining up. In the case that there's a function longer than `companyAlternativeNames`, we'd have to go through and align everything again. 

We can go ahead and keep it since you spent the time making them line up, but in the future I wouldn't bother with it.

Reply:
the prettier extension, in vscode did this on its own.

Reply:
oh neat!